src/modules/fw_att_control/ecl_roll_controller.cpp

```diff
@@ -46,14 +46,17 @@
 float ECL_RollController::control_attitude(const struct ECL_ControlData &ctl_data)
 {
 	/* Do not calculate control signal with bad inputs */
-	if (!(PX4_ISFINITE(ctl_data.roll_setpoint) && PX4_ISFINITE(ctl_data.roll))) {
+	if (!(PX4_ISFINITE(ctl_data.roll_setpoint) &&
+	      PX4_ISFINITE(ctl_data.roll))) {
+
+		PX4_WARN("not controlling roll");
```

Comment:
I don't think this is useful and it's using quite some flash space.
```suggestion
```

Reply:
Did for all three modules. (The warning was originally only there for the pitch controller, so I had copied it to the others. Now none of them will show it.)